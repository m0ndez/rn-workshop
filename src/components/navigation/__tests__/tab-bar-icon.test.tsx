import React from "react";
import { render } from "@testing-library/react-native";

import { TabBarIcon } from "../tab-bar-icon";

describe("TabBarIcon component", () => {
  it("renders correctly", () => {
    const { toJSON } = render(
      <TabBarIcon name="home" size={24} style={{ color: "red" }} />
    );

    expect(toJSON()).toMatchSnapshot();
  });
});
