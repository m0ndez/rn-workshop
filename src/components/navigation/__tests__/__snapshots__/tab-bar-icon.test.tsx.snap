// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabBarIcon component renders correctly 1`] = `<Text />`;
